charmcraft/cmdbase.py

```diff
@@ -53,3 +56,22 @@ def include_format_option(self, parser):
             choices=[JSON_FORMAT],
             help=FORMAT_HELP_STR,
         )
+
+    def _check_config(self, config_file: bool = False, bases: bool = False) -> None:
+        """Check if requested config content exists.
+
+        - config_file: request a valid 'charmcraft.yaml' file.
```

Comment:
The meaning of "request" isn't clear for either of these variables, it hints that something is returned. 

I would reword this to something like "If true, check if a valid charmcraft.yaml file exists."